drops.go

```diff
@@ -0,0 +1,56 @@
+package helix
+
+// GetDropEntitlementsParams ...
+type GetDropEntitlementsParams struct {
+	EntitlementID string `query:"id"`
```

Comment:
Please change `EntitlementID` to `ID` to be consistent with other structs in this library.

Reply:
Thanks for pointing it out - fixed now.